axelrod/actions.py

```diff
@@ -1,10 +1,14 @@
+from typing import NewType
+
+C = NewType('Action', str)
```

Comment:
I think you probably want `Action = NewType('Action', str)` and to declare C and D as actions. Unfortunately it's not easy to do so without the Py3.6 additions to type hints. But I think this would still work for `flip_action`.